apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -211,6 +221,84 @@ export class TeacherFeedback extends Component {
     return feedbackUnchanged;
   };
 
+  getFriendlyDate(feedbackSeen) {
+    const today = new Date();
+    const dateFeedbackSeen = new Date(feedbackSeen);
+
+    //javascript months begin counting at 0
+    const [todayM, todayD, todayY] = [
+      today.getMonth() + 1,
+      today.getDate(),
+      today.getFullYear()
+    ];
+
+    const [m, d, y] = [
+      dateFeedbackSeen.getMonth() + 1,
```

Comment:
Tiny NIT: Is this +1 and the one in 230 still needed? It looks like m and todayM are only used to compare against each other.

Reply:
Really good point - no longer relevant now that I'm using moment.js